docs/skopeo-standalone-sign.1.md

```diff
@@ -25,6 +25,10 @@ $ skopeo standalone-sign busybox-manifest.json registry.example.com/example/busy
 $
 ```
 
+## NOTES
+
+This command is intended for use with local signatures, such as OpenPGP, as per containers-signature(5). Furthermore, this command does **not** interact with the artifacts generated by Docker Content Trust (DCT). For more information, please see [containers-signature(5)](https://github.com/containers/image/blob/main/docs/containers-signature.5.md).
```

Comment:
Ah, OK, so I can indicate that OpenPGP is an example, and that other signature formats may be added in the future

Reply:
I’m not sure spelling these tentative plans out is all that useful either.

Or do you think OpenPGP is important because it allows identifying the signature file format, or something like that?

Reply:
TBH I don't mind either way, I just used OpenPGP 'cos that's what I've been using for my own tests here - `gpg (GnuPG) 2.2.19`